webapp/src/pages/Settings.tsx

```diff
@@ -637,55 +647,83 @@ const Settings: React.FC = () => {
   );
 
   return (
-    <Box height="100%" display="flex" flexDirection="column">
-      <Paper
-        variant="outlined"
-        sx={{
-          flex: 1,
-          padding: 2,
-          overflow: "auto",
-          [`& .${formControlLabelClasses.labelPlacementStart}`]: {
-            justifyContent: "flex-end",
-            marginLeft: 0,
-          },
-          [`& .${formGroupClasses.root}`]: { marginX: 2, marginBottom: 2 },
-          [`& .fixedWidthInput .${inputClasses.root}`]: { width: "12ch" },
-          [`& .${inputClasses.input}`]: { fontSize: 14, padding: 0 },
-          [`& .${inputLabelClasses.root}`]: { fontWeight: "bold" },
-        }}
-      >
-        <Typography variant="subtitle1" marginBottom={3}>
-          View and edit certain fields from your config file. Once your changes
-          are saved, expect some delays for recomputing the affected tasks.
-        </Typography>
-        <AccordionLayout
-          name="Project Configuration"
-          description="View the fields that define the dataset to load in Azimuth."
-          link="reference/configuration/project/"
-        >
-          {getProjectConfigSection()}
-        </AccordionLayout>
-        <AccordionLayout
-          name="Model Contract Configuration"
-          description="View and edit some fields that define the ML pipelines and the metrics."
-          link="reference/configuration/model_contract/"
-        >
-          {getModelContractConfigSection()}
-        </AccordionLayout>
-        <AccordionLayout
-          name="Analyses Customization"
-          description="Enable or disable some analyses and edit corresponding thresholds."
-          link="reference/configuration/analyses/"
+    <>
+      <DialogTitle id="config-dialog-title">
+        <Box display="flex" justifyContent="space-between">
+          <Typography variant="subtitle1">
+            View and edit certain fields from your config file. Once your
+            changes are saved, expect some delays for recomputing the affected
+            tasks.
+          </Typography>
+          <Tooltip title="close" placement="bottom">
+            <IconButton
+              size="small"
+              color="primary"
+              sx={{ padding: 0 }}
+              onClick={() => {
+                if (isUpdatingConfig) {
+                  window.alert(CONFIG_UPDATE_MESSAGE);
+                } else if (
```

Comment:
Alternatively, we could disable the button while the config is updating. What do you think?
```suggestion
              disabled={isUpdatingConfig}
              onClick={() => {
                if (
```

Reply:
Sure. good one. How about we disable the discard button too?

Reply:
Sounds good to me!